CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Add support for finding Unity in the Hub alternate install path ([#2015](https://github.com/getsentry/sentry-unity/pull/2015))
+
```

Comment:
```suggestion
```
The changelog strictly contains user facing changes. Any repo-maintence or utility/CI changes can be annotated with `#skip-changelog` in the issue description to silence `Danger`.

Reply:
Ah, gotcha, thanks!